src/olympia/blocklist/mlbf.py

```diff
@@ -207,13 +209,21 @@ def hash_filter_inputs(cls, input_list):
             for (guid, version) in input_list
         ]
 
-    def filter_path(self, _block_type: BlockType = BlockType.BLOCKED):
-        return self.storage.path('filter')
+    def filter_path(self, block_type: BlockType):
+        # TODO: explain / test
+        if block_type == BlockType.BLOCKED:
+            return self.storage.path('filter')
```

Comment:
Why do we need to special case this path? Backward compatibility?

Reply:
Correct. Existing filters in the file system are going to have that name.